IaC/AKS/cluster/charts/fluentbit/templates/config.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "fluentbit.fullname" . }}
+data:
+  fluent-bit.conf: |
+    [SERVICE]
+        Flush         1
+        Log_Level     info
+        Daemon        off
+        Parsers_File  parsers.conf
+        HTTP_Server   On
+        HTTP_Listen   0.0.0.0
+        HTTP_Port     2020
+    @INCLUDE input-kubernetes.conf
+    @INCLUDE filter-kubernetes.conf
+    @INCLUDE output.conf
+
+  output.conf: |
+    [OUTPUT]
+        Name            azure
+        Match           kube.var.log.containers.webv*.*
+        Customer_ID     ${LOG_ID}
+        Shared_Key      ${LOG_SHARED_KEY}
+        Log_Type        webv${LOG_SUFFIX}
```

Comment:
since we forked webv to loderunner, we should move away from using webv - either loderunner or l8r